dev_docs/getting_started/setting_up_a_development_env.mdx

```diff
@@ -46,6 +46,13 @@ Then, install the latest version of yarn using:
 npm install -g yarn
 ```
 
+If you already have Yarn installed, make sure its version is 1.22.19 or newer. Versions 2+ of Yarn are incompatible with this project.
+
+In order to set your previously installed Yarn to a compatible version (for example 1.22.19) run
+```sh
+yarn set version 1.22.19
+```
+
 Finally, bootstrap Kibana and install all of the remaining dependencies:
```

Comment:
```suggestion
. Bootstrap {kib}, then install the remaining dependencies:
```

Reply:
Same doubt here. `{kib}` placeholders are only found on *.asciidocs. Should I ignore and resolve this as well?

Reply:
Yes, please ignore :)